src/System.Data.SqlClient/ref/project.json

```diff
@@ -5,7 +5,8 @@
     "System.IO": "4.0.0",
     "System.Runtime": "4.0.0",
     "System.Threading.Tasks": "4.0.0",
-    "System.Xml.ReaderWriter": "4.0.0"
+    "System.Xml.ReaderWriter": "4.0.0",
+	"System.Collections.NonGeneric": "4.0.0",
```

Comment:
> would it be OK to keep it 4.0.10.0 because it is just an intermittent change before an official "release"?
 Add a line note

Yes It would be OK to keep the same version before the release.